optaweb-vehicle-routing-docs/src/main/asciidoc/modules/proc-vrp-build.adoc

```diff
@@ -0,0 +1,28 @@
+[id="proc-vrp-build_{context}"]
+
+= Building OptaWeb Vehicle Routing
+COMMENT: This seems incomplete or out of context. Flagging to review in the context of the entire guide.
```

Comment:
We don't ship the binary build of the application (optaweb-...-standalone.jar) in the **product** artifacts, just the sources. So users that download RHDM's reference-implementation.zip will only have sources of OptaWeb and need to build it from source. They can't use run.sh.

Reply:
OK. I'll need to work on this later.